code/modules/keybindings/bindings_mob.dm

```diff
@@ -0,0 +1,77 @@
+// Technically the client argument is unncessary here since that SHOULD be src.client but let's not assume things
+// All it takes is one badmin setting their focus to someone else's client to mess things up
+// Or we can have NPC's send actual keypresses and detect that by seeing no client
+
+/mob/key_down(_key, client/user)
```

Comment:
Noting here that we should convert all of this to a decl system after this PR, I know you're splitting them up but it's worth mentioning briefly.